src/analysis/locally_convex/bounded.lean

```diff
@@ -125,3 +127,39 @@ is_bounded_of_bounded_iff _
 end normed_field
 
 end bornology
+
+section uniform_add_group
+
+variables [nondiscrete_normed_field 𝕜] [add_comm_group E] [module 𝕜 E]
+variables [uniform_space E] [uniform_add_group E] [has_continuous_smul 𝕜 E]
+variables [regular_space E]
+
+lemma totally_bounded.is_vonN_bounded {s : set E} (hs : totally_bounded s) :
+  bornology.is_vonN_bounded 𝕜 s :=
+begin
+  rw totally_bounded_iff_subset_finite_Union_nhds_zero at hs,
+  intros U hU,
+  have h : filter.tendsto (λ (x : E × E), x.fst + x.snd) (𝓝 (0,0)) (𝓝 ((0 : E) + (0 : E))) :=
+  continuous_iff_continuous_at.mp has_continuous_add.continuous_add (0,0),
```

Comment:
I didn't try, but shouldn't this be something like `continuous_add.continuous_at (0,0)`?

Reply:
There is `tendsto_add`, but no `tendsto_add_zero`.